src/main/kotlin/com/bugsnag/android/gradle/BugsnagPlugin.kt

```diff
@@ -282,6 +282,18 @@ class BugsnagPlugin : Plugin<Project> {
             if (uploadSourceMapProvider != null) {
                 variant.register(project, uploadSourceMapProvider, reactNativeEnabled)
             }
+
+            project.tasks.named("installRelease").configure {
+                if (reactNativeEnabled) {
+                    project.logger.lifecycle(
+                        "Bugsnag: JS sourcemaps and JVM/NDK mapping files are " +
+                            "not uploaded when calling react-native run-android. If you want to upload JS " +
+                            "sourcemaps, you should generate a release APK using ./gradlew assembleRelease" +
+                            " or ./gradlew bundleRelease instead. See the React Native docs for further info: " +
+                            "https://reactnative.dev/docs/signed-apk-android#generating-the-release-apk"
```

Comment:
The latter still mentions JS sourcemaps specifically - suggest just following it with the recommendation.

Also, expanded the description of the react-native CLI.

```suggestion
                        "Bugsnag: JS sourcemaps and JVM/NDK mapping files are not uploaded when " +
                            "using the react-native CLI (e.g. react-native run-android --variant=Release). " +
                            "You should generate a release APK using ./gradlew assembleRelease or " +
                            "./gradlew bundleRelease instead. See the React Native docs for further info: " +
                            "https://reactnative.dev/docs/signed-apk-android#generating-the-release-apk"
```